sql/sys_vars.cc

```diff
@@ -5593,6 +5617,12 @@ static Sys_var_rpl_filter Sys_replicate_do_db(
        "actual names of table(s) being updated are checked.",
        PRIV_SET_SYSTEM_GLOBAL_VAR_REPLICATE_DO_DB);
 
+static Sys_var_binlog_filter Sys_binlog_do_db(
```

Comment:
Just a thought, it'd be great eventually to have the class' method to work in `mysqld_get_one_option` that **actually** "populates" the instance.